### ISSUE TITLE
Can's set zero precision to timestamp

### ISSUE THREAD

**Issue by 6aKa** [knex/knex#4784]
# Environment

Knex version: Knex Local version: 0.95.12
Database + version: psql (PostgreSQL) 13.4
OS: MacOS Monterey 12.0.1 (21A559)

# Bug
In migration create field
`t.timestamp('confirmed_at', {useTz: false, precision: 0});`
```
     Column      |            Type             | Collation | Nullable |      Default      | Storage  | Stats target | Description
-----------------+-----------------------------+-----------+----------+-------------------+----------+--------------+-------------
 confirmed_at    | timestamp without time zone |           |          |                   | plain    |              |
```

Expected `timestamp(0) without time zone`

From PostgreSQL documentation
```
time, timestamp, and interval accept an optional precision value p which specifies the number of fractional digits retained in the seconds field. By default, there is no explicit bound on precision. The allowed range of p is from 0 to 6 for the timestamp and interval types.
```

**Comment by OlivierCavadenti** [comment]
Thanks for the report.

I check quickly and it's seems the problem occurs with precision = 0 because of that check during the column compile phase :

`precision = precision ? '(' + precision + ')' : '';`

I will make a fix quickly.

**Comment by kibertoad** [comment]
Released in 1.0.1.
